browser/src/view/components/Button/Button.scss

```diff
@@ -1,27 +1,26 @@
 @use '../../../styles/common/variables' as *;
 
-.btn-primary {
-  height: 48px;
-  width: 48px;
+.button {
+  font-family: 'Inter';
+  font-weight: 500;
```

Comment:
We should probably define different font weights in our variables and use the variables here.